src/v2/guide/computed.md

```diff
@@ -300,6 +295,6 @@ var watchExampleVM = new Vue({
 </script>
 {% endraw %}
 
-In this case, using the `watch` option allows us to perform an asynchronous operation (accessing an API), limit how often we perform that operation, and set intermediary states until we get a final answer. None of that would be possible with a computed property.
+Trong trường hợp này, sử dụng `watch` cho phép chúng ta thực hiện những tính toán không đồng bộ (ví dụ: truy cập tới một API) và giới hạn những tính toán đó, sau đó giá trị sẽ được gán ngay lập tức vào state khi có được đáp án cuối cùng. Những điều trên sẽ không thực hiện được nếu bạn sử dụng thuộc tính computed.
```

Comment:
* "limit how often we perform that operation" mà dịch thành "giới hạn những tính toán đó" thì không chỉ thiếu mà còn sai ý
* "set intermerdiary states" là "gán trạng thái trung gian" chứ không phải "gán ngay lập tức vào state" gì cả. Nguyên cụm này dịch sai. Ý người ta là: Dùng `watch` cho phép chúng ta set một trạng thái trung gian (như là bật cờ loading, đổi text "now loading," chạy animation…) _cho đến khi_ nhận được phản hồi (không phải đáp ứng).
* "Những điều trên sẽ không thực hiện được nếu bạn sử dụng thuộc tính computed." -> "Nếu dùng computed property bạn sẽ không làm được những chuyện này."